pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -489,6 +489,50 @@ func newCertRotationController(
 	)
 	ret.certRotators = append(ret.certRotators, certRotator)
 
+	certRotator = certrotation.NewCertRotationController(
+		"MachineSystemAdminClient",
+		certrotation.SigningRotation{
+			Namespace:              operatorclient.OperatorNamespace,
+			Name:                   "machine-system-admin-signer",
+			Validity:               10 * 365 * defaultRotationDay,
```

Comment:
no need to be this long.  One year should be enough for everyone.

Reply:
This is long lived so people who loose their kubeconfig can just copy it over.

Reply:
also `localhost-recovery.kubeconfig` need long lived